service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/ServiceCenterTask.java

```diff
@@ -30,16 +34,25 @@
 
   private int interval;
 
+  private int checkTimes;
+
+  private AtomicLong failedTimes = new AtomicLong();
```

Comment:
this propertie should be named as consecutiveXXX

Reply:
done